src/admin/components/forms/Error/index.tsx

```diff
@@ -9,7 +9,7 @@ const baseClass = 'field-error';
 const Error: React.FC<Props> = (props) => {
   const {
     showError = false,
-    message = 'Please complete this field.',
+    message,
```

Comment:
Is there a replacement for this message? Couldn't find it in translations

Reply:
No. This was a default message and as far as I could tell it wasn't being used. Everywhere this component error was used we had already included a message in the props.

I deleted this default text to simplify getting translations done.